recipes/con-duct/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "con-duct" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/con_duct-{{ version }}.tar.gz
+  sha256: cb65c5ff4590780a048000e72bf3cbcca250f282257bbd50fea140670ae9f390
+
+build:
+  skip: true  # [win or py<38 or osx]
```

```suggestion
  noarch: python 
```
Looks like it can be noarch

---

@xhochy unfortunately, con-duct doesn't work properly on windows or osx, as it depends on the linux implementation of `ps`  with session ids.

---

Then please add a __glibc constrain here as a way to marknit linux-only but make it noarch.